### PR TITLE
Feature/jhipster 5.7.2

### PR DESCRIPTION
**Pull request by vkirichenko**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -141,6 +142,3 @@ Desktop.ini
 # ESLint
 ######################
 .eslintcache
-
-src/main/lep-sandbox
```
Comment (amedvedchuk) [inline .gitignore]: please keep this directory ignored to connect LEP scripts with symlink.

Comment (vkirichenko) [reply .gitignore]: done

File: settings.gradle [hunk 2]
```diff
@@ -1 +1 @@
-rootProject.name = 'xm-uaa'
+rootProject.name = 'uaa'
```
Comment (amedvedchuk) [inline settings.gradle]: I think we should not change project name. Pls check if it was not mistake.

File: src/main/java/com/icthh/xm/uaa/UaaApp.java [hunk 3]
```diff
@@ -78,49 +76,58 @@ private static void initContexts() {
         MdcUtils.putRid(MdcUtils.getRid() + "::" + superKey.getValue());
     }
 
-    @PreDestroy
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/uaa/UaaApp.java]: Please do not remove this log. Sometimes it helps to identify exact point when application was shutdown.

File: src/main/resources/config/application-dev.yml [hunk 4]
```diff
@@ -3,66 +3,87 @@
...
-        url: jdbc:h2:mem:uaa;DB_CLOSE_DELAY=-1
+        url: jdbc:h2:mem:uaa;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
         username: uaa
         password:
+        hikari:
+            auto-commit: false
     h2:
         console:
-            enabled: true
+            enabled: false
```
Comment (amedvedchuk) [inline src/main/resources/config/application-dev.yml]: I thin it is good idea to have H2 web console in development mode.

File: src/main/resources/config/application-dev.yml [hunk 5]
```diff
@@ -3,66 +3,87 @@
...
-            #hibernate.cache.hazelcast.use_lite_member: true
+    liquibase:
+        contexts: dev
+        enabled: true
     mail:
-            host:
-            port:
-            username:
-            password:
-            properties:
```
Comment (amedvedchuk) [inline src/main/resources/config/application-dev.yml]: Need to check if absence of these properties will not break env variables injected into docker container. 
@ykruchko what do you think?

File: src/main/resources/config/application-dev.yml [hunk 6]
```diff
@@ -72,64 +93,38 @@ spring:
...
     security:
         client-authorization:
-            client-id:
-            client-secret:
+            client-id: internal
+            client-secret: internal
     mail: # specific JHipster mail property, for standard properties see MailProperties
-        from: <tenantname>@xm-online.com
-        base-url: http://127.0.0.1:9999
+        from: uaa@localhost
```
Comment (amedvedchuk) [inline src/main/resources/config/application-dev.yml]: As far as i know there is some dependency in mail sending code which should use <tenantname> placeholder. So please check if it is not break the functionality.

File: src/main/resources/config/application-prod.yml [hunk 7]
```diff
@@ -13,10 +13,13 @@
...
+logging:
+    level:
+        ROOT: INFO
+        com.icthh.xm.uaa: INFO
+        io.github.jhipster: INFO
+
 spring:
-    profiles:
-        active: prod
-        include: swagger
```
Comment (amedvedchuk) [inline src/main/resources/config/application-prod.yml]: Suggest to check if removing swagger profile will not prevent from exposing swagger at all.

File: src/main/resources/config/application-prod.yml [hunk 8]
```diff
@@ -102,21 +114,22 @@ jhipster:
...
+            client-secret: internal
+        authentication:
+            jwt:
+                # Access Token is valid for 5 mins
+                token-validity-in-seconds: 300
+                # Refresh Token is valid for 7 days
+                token-validity-in-seconds-for-remember-me: 252000
     mail: # specific JHipster mail property, for standard properties see MailProperties
-        from: <tenantname>@xm-online.com
+        from: uaa@localhost
```
Comment (amedvedchuk) [inline src/main/resources/config/application-prod.yml]: As far as i know there is some dependency in mail sending code which should use placeholder. So please check if it is not break the functionality.

File: src/main/resources/config/application-dev.yml (hunk 5 above)
Comment (vkirichenko) [reply src/main/resources/config/application-dev.yml]: i will check

File: src/main/resources/config/application-prod.yml (hunk 8 above)
Comment (vkirichenko) [reply src/main/resources/config/application-prod.yml]: fixed

File: src/main/resources/config/application-dev.yml (hunk 6 above)
Comment (vkirichenko) [reply src/main/resources/config/application-dev.yml]: done

File: src/main/resources/config/application-dev.yml (hunk 4 above)
Comment (vkirichenko) [reply src/main/resources/config/application-dev.yml]: done

File: src/main/java/com/icthh/xm/uaa/UaaApp.java (hunk 3 above)
Comment (vkirichenko) [reply src/main/java/com/icthh/xm/uaa/UaaApp.java]: done

File: settings.gradle (hunk 2 above)
Comment (vkirichenko) [reply settings.gradle]: fixed

File: src/main/resources/config/application-prod.yml (hunk 7 above)
Comment (vkirichenko) [reply src/main/resources/config/application-prod.yml]: returned back

File: src/main/resources/config/application-dev.yml (hunk 5 above)
Comment (vkirichenko) [reply src/main/resources/config/application-dev.yml]: everything ok

**Pull request closed without merging**